docs/source/features.rst

```diff
@@ -365,14 +365,16 @@ For instance:
 Obviously, you should only define default values that are appropriate for
 a given file system implementation. INI files only support string values.
 
-Alternatively, you can provide overrides with environment variables of
-the style ``FSSPEC_{protocol}_{kwargname}=value``.
+Alternatively, you can provide overrides with environment variables of the style
+``FSSPEC_{protocol}=<json_dict_value>`` and
+``FSSPEC_{protocol}_{kwargname}=<string_value>``.
 
 Configuration is determined in the following order, with later items winning:
 
 #. the contents of ini files, and json files in the config directory, sorted
    alphabetically
```

Comment:
Not really a part of this PR but...

Every time I read this first item, I find myself thinking "where the heck does fsspec think the 'config directory' is"?  And then I have to go search and find https://github.com/fsspec/filesystem_spec/blob/05b886ab23ae9de7d101d0621476ba61d66df29d/fsspec/config.py#L6-L7

Could we lump in this change here:
```suggestion
#.  ini and json files in the config directory (``FSSPEC_CONFIG_DIRECTORY`` or $HOME/.config/fsspec/), sorted
   lexically by filename
```

Reply:
Feel free to repeat, yes